doc/source/reference/random/multithreading.rst

```diff
@@ -12,31 +12,27 @@ these requirements.
 This example makes use of Python 3 :mod:`concurrent.futures` to fill an array
 using multiple threads.  Threads are long-lived so that repeated calls do not
 require any additional overheads from thread creation. The underlying
-BitGenerator is `PCG64` which is fast, has a long period and supports
-using `PCG64.jumped` to return a new generator while advancing the
-state. The random numbers generated are reproducible in the sense that the same
-seed will produce the same outputs.
+BitGenerator is `PCG64` is fast, has a long period and is seeded by quasi-independent
+seeds of a `~SeedSequence`.
+
+The random numbers generated are reproducible in the sense that the same
+seed will produce the same outputs, given that the number of threads does not change.
 
 .. code-block:: ipython
 
-    from numpy.random import Generator, PCG64
+    from numpy.random import Generator, PCG64, SeedSequence
     import multiprocessing
     import concurrent.futures
     import numpy as np
 
     class MultithreadedRNG:
         def __init__(self, n, seed=None, threads=None):
-            bg = PCG64(seed)
             if threads is None:
                 threads = multiprocessing.cpu_count()
             self.threads = threads
 
-            self._random_generators = [Generator(bg)]
-            last_bg = bg
-            for _ in range(0, threads-1):
-                new_bg = last_bg.jumped()
-                self._random_generators.append(Generator(new_bg))
-                last_bg = new_bg
+            seq = SeedSequence(seed)
+            self._random_generators = [Generator(PCG64(s)) for s in seq.spawn(threads)]
```

Comment:
Just a style thing: consider breaking up the list comp so it stays under 80-char per line:

```python
self._random_generators = [
    Generator(PCG64(s)) for s in seq.spawn(threads)
]
```

Reply:
Since we've changed from using `.jumped()` to the `BitGenerator`-agnostic solution, I would recommend using `default_rng(s)` instead of `Generator(PCG64(s))`.

Reply:
Thanks for the input. I'll change it accordingly.